examples/seq2seq/europal.py

```diff
@@ -25,7 +26,7 @@ def split_sentence(s):
 
 def open_file(path):
     if path.endswith('.gz'):
-        return gzip.open(path)
+        return gzip.open(path, 'rt', sys.getdefaultencoding())
```

Comment:
The encoding must depends on data, not on the system. So, please add encoding argument to `open_file` and use 'utf-8' directly because file this example uses here is encoded in utf-8.